src/events/guildMemberAdd.ts

```diff
@@ -87,14 +87,14 @@ export default class extends Event {
 
         if (guildSettings.hibye.welcome.dmEnabled) {
           const dmChannel = await member.user.getDMChannel()
-          if (embed) await dmChannel.createMessage({ embed })
-          else await dmChannel.createMessage(transformed)
+          if (embed) await dmChannel.createMessage({ embed }).catch(() => undefined)
+          else await dmChannel.createMessage(transformed).catch(() => undefined)
         }
         if (guildSettings.hibye.welcome.channelID) {
           const welcomeChannel = guild.channels.get(guildSettings.hibye.welcome.channelID)
           if (welcomeChannel && welcomeChannel instanceof TextChannel) {
-            if (embed) welcomeChannel.createMessage({ embed })
-            else welcomeChannel.createMessage(transformed)
+            if (embed) welcomeChannel.createMessage({ embed }).catch(() => undefined)
```

Comment:
```suggestion
            if (embed) await welcomeChannel.createMessage({ embed })
```
Remove the .catch for await instead if it is awaited it will be caught by the try catch